lib/aggregation/accumulator/src/test/previous-month-test.js

```diff
@@ -0,0 +1,216 @@
+'use strict';
+
+// Usage accumulator service.
+/* eslint-disable nodate/no-moment, nodate/no-new-date, nodate/no-date */
+
+const _ = require('underscore');
+const request = require('abacus-request');
+const cluster = require('abacus-cluster');
+const dbclient = require('abacus-dbclient');
+
+const extend = _.extend;
+const map = _.map;
+
+const debug = require('abacus-debug')('abacus-usage-accumulator-previous-month-test');
+
+/* eslint handle-callback-err: 0 */
+
+const testEnv = {
+  db: process.env.DB
+};
+
+describe('abacus-usage-accumulator', () => {
+  let clock;
+  let accumulator;
+  let postSpy;
+  let server;
+
+  const mockCluster = () => {
```

Comment:
We can move the mock functions into separate util and call them from this test, as well as the main test suite `test.js`

Reply:
After a brief discussion, we have decided that the value of this change would not justify the time it would need.